runner/runs/quad_single_gridsearch.py

```diff
@@ -17,7 +17,8 @@
 
 _experiment = Experiment(
     'quads_gridsearch',
-    'python -m run_algorithm --env=quadrotor_single --train_for_env_steps=100000 --algo=APPO --gamma=0.99 --use_rnn=False --num_workers=36 --num_envs_per_worker=2 --num_policies=1 --rollout=32 --recurrence=32 --benchmark=False --with_pbt=False --ppo_clip_ratio=0.05',
+    'python -m run_algorithm --env=quadrotor_single --train_for_env_steps=100000 --algo=APPO --gamma=0.99 --use_rnn=False --num_workers=8 --num_envs_per_worker=2 --num_policies=1 --rollout=32 --recurrence=32 --benchmark=False --with_pbt=False --ppo_clip_ratio=0.05',
```

Comment:
How about resume to the previous version? Since I don't think change `num_workers`  from 36 to 8 make sense.